docs/repositories/commits.md

```diff
@@ -116,6 +130,8 @@ The **Diff** tab displays the differences in each file that was changed in the {
 -   **Green**: New version of a changed line with added characters highlighted in bright green
 -   **Bright green**: New line
```

Comment:
these, and the text before, is wrong

we changed how we show the diff; before this diff was showing just changes (no issues, no nothing) and it was using red, yellow, green and bright green to "show changes", but now we just use a split view, deletions and additions are shown with opacity and a blue line in the margin

Reply:
Thanks, good catch. Updating